po/fr.po

```diff
@@ -11540,13 +11541,13 @@ msgstr "Paramètre --name manquant"
 
 #: gnucash/gnucash-cli.cpp:185
 msgid "Unknown report command '{1}'"
-msgstr ""
+msgstr "Commande '{1}' inconnue"
 
 # po/guile_strings.txt:277
 #: gnucash/gnucash-cli.cpp:191
 #, fuzzy
 msgid "Missing command or option"
-msgstr "C'est une option de couleur"
+msgstr "Commande ou option manquante"
```

Comment:
@Lisapple If you translate a fuzzy  message, remove the fuzzy flag. In weblate AFAIK this will happen, when you change the state from `Needs editing` to `Waiting for review`.

This is not the only occurrence.